tracy-client/src/gpu.rs

```diff
@@ -0,0 +1,163 @@
+use std::sync::atomic::{AtomicU16, AtomicU8, Ordering};
+
+use crate::Client;
+
+#[repr(C)]
+#[derive(Debug, Copy, Clone)]
+struct ___tracy_gpu_new_context_data {
+    gpu_time: i64,
+    period: f32,
+    context: u8,
+    flags: u8,
+    type_: u8,
+}
+
+#[repr(u8)]
+///
+pub enum GpuContextType {
```

Comment:
Unfortunately I don't see an easy way to, considering it's only defined in the TracyQueue.hpp file

Reply:
Ah, I think I meant that that this would be a contribution to the tracy project to add the enum definition to the `TracyC.h` file. In my past experience the author of the project is receptive to changes like these.

(Of course I wouldn’t want to block landing of this PR on a new release of Tracy, so it is fine to have this here as-is for the time being)